src/main/java/com/iota/iri/crypto/batched/BCTCurl.java

```diff
@@ -0,0 +1,121 @@
+package com.iota.iri.crypto.batched;
```

Comment:
this package should be submitted as a separate PR imo, as it can stand alone and be reviewed as such.

Reply:
it's already in a separate PR https://github.com/iotaledger/iri/pull/1372